tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,84 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`conditional-expression.js 1`] = `
+<div>
+  {isCool ? "cool" : "very nice"}
+</div>;
+
+// should get formatted JSX style
+const a = something ? "string" : <div />;
+
+// should not get formatted JSX style
+const a = something ? "string" : "other string";
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    "why"
+  ) : yes ? (
+    "it"
+  ) : is ? (
+    "!"
+  ) : (
+    ":)"
+  )}
+</div>;
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    <span>why</span>
+  ) : yes ? (
+    <span>it</span>
+  ) : is ? (
+    <span>!</span>
+  ) : (
+    <span>:)</span>
+  )}
+</div>;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+  {isCool ? (
```

Comment:
Update: I tried to group this properly so that the example in question would stay on one line, but ran into issues like I described (where one half of the conditional expression is multiline, and the other is not). This looked particularly wonky with chained conditional expressions. I can go into more detail, but the tl;dr is I couldn't get it working, and was starting to build more and more complex rules re: ConditionalExpression in JSX, that made it hard to predict what the code was going to do.

It may be worth revisiting this logic in the future, but in the meantime, this PR seems to be an improvement over the current implementation, so I'd like to punt on that improvement for now if possible.